### PR TITLE
Vectorize query HTTP endpoint changes

### PR DESCRIPTION
**Pull request by vy-ton**

### CODE REVIEW

File: data/changelogs/vectorize.yaml [hunk 1]
```diff
@@ -2,6 +2,34 @@
 link: "/vectorize/changelog/"
 productName: Vectorize
 entries:
+  - publish_date: "2024-01-17"
+    title: HTTP API query vectors request and response format change
+    description: |-
+      Vectorize `/query` HTTP endpoint has the following changes:
+      - `returnVectors` request body property is deprecated in favor of `returnValues` and `returnMetadata` properties
```
Comment (Maddy-Cloudflare) [inline data/changelogs/vectorize.yaml]:
```suggestion
      - `returnVectors` request body property is deprecated in favor of `returnValues` and `returnMetadata` properties.
```

File: data/changelogs/vectorize.yaml [hunk 2]
```diff
@@ -2,6 +2,34 @@
...
+      - Response format has changed to the below format to match [Workers API change](/workers/configuration/compatibility-dates/#vectorize-query-with-metadata-optionally-returned)
```
Comment (Maddy-Cloudflare) [inline data/changelogs/vectorize.yaml]:
```suggestion
      - Response format has changed to the below format to match [Workers API change]:(/workers/configuration/compatibility-dates/#vectorize-query-with-metadata-optionally-returned)
```

**Review by Maddy-Cloudflare — COMMENTED** [review]
You may want to pull the latest changes as `/vectorize/changelog/` is now `/vectorize/platform/changelog/`.

**Review by ndisidore — APPROVED** [review]
Looks great! Good shout linking to the binding change - helps with the consistency story :+1:

File: data/changelogs/vectorize.yaml [hunk 3]
```diff
@@ -2,6 +2,34 @@
...
+
+      ```json
+      {
+        "result": {
+          "count": 0,
```
Comment (ndisidore) [inline data/changelogs/vectorize.yaml]: nit: this would be `1` if there's one item in the `matches` array

**Review by elithrar — APPROVED** [review]

**Pull request merged**